Integration Tests fail in strongbox-web-core due to message mismatch with non english locale

# Bug Description

Integration tests in strongbox-web-core fail when run on a system with non english locale.

# Steps To Reproduce

1. Clone repo
2. Set system locale to something other than english (tested on german)
3. Build repo with `mvn clean install -Dintegration.tests`

# Expected Behavior
Tests should not be affected by locale of the building system

# Environment
* Strongbox Version: 1.0-SNAPSHOT
* OS: Windows 10 x64 - German
* Maven version (please provide the full output of `mvn --version`):

`Apache Maven 3.6.1 (d66c9c0b3152b2e69ee9bac180bb8fcc8e6af555; 2019-04-04T21:00:29+02:00)
Maven home: C:\Program Files\JetBrains\IntelliJ IDEA 2019.2\plugins\maven\lib\maven3
Java version: 1.8.0_202, vendor: Oracle Corporation, runtime: C:\Program Files\Java\jre1.8.0_202
Default locale: de_AT, platform encoding: Cp1252
OS name: "windows 10", version: "10.0", arch: "amd64", family: "windows"`

# Proposed Solution 
Implement fix similar to [Issue 1466/Pull 1467](https://github.com/strongbox/strongbox/pull/1467)

# Known Workarounds
Change system locale to english

# Useful Links <!-- Remove if unrelated -->

* [Similar Issue](https://github.com/strongbox/strongbox/issues/1466)

Is there are particular integration test that fails or all of them. When I run, I get this one failing ` org.carlspring.strongbox.controllers.layout.npm.NpmArtifactControllerTest`. However it does not name specific test(s) that are failing...
```
org.carlspring.strongbox.controllers.layout.npm.NpmArtifactControllerTest
[ERROR]         at org.apache.maven.plugin.surefire.booterclient.ForkStarter.fork(ForkStarter.java:669)
[ERROR]         at org.apache.maven.plugin.surefire.booterclient.ForkStarter.run(ForkStarter.java:282)
[ERROR]         at org.apache.maven.plugin.surefire.booterclient.ForkStarter.run(ForkStarter.java:245)
[ERROR]         at org.apache.maven.plugin.surefire.AbstractSurefireMojo.executeProvider(AbstractSurefireMojo.java:1183)
[ERROR]         at org.apache.maven.plugin.surefire.AbstractSurefireMojo.executeAfterPreconditionsChecked(AbstractSurefireMojo.java:1011)
[ERROR]         at org.apache.maven.plugin.surefire.AbstractSurefireMojo.execute(AbstractSurefireMojo.java:857)
[ERROR]         at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:137)
[ERROR]         at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:208)
[ERROR]         at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:154)
[ERROR]         at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:146)
[ERROR]         at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:117)
[ERROR]         at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:81)
[ERROR]         at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:56)
[ERROR]         at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:128)
[ERROR]         at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:305)
[ERROR]         at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:192)
[ERROR]         at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:105)
[ERROR]         at org.apache.maven.cli.MavenCli.execute(MavenCli.java:954)
[ERROR]         at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:288)
[ERROR]         at org.apache.maven.cli.MavenCli.main(MavenCli.java:192)
[ERROR]         at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[ERROR]         at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[ERROR]         at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[ERROR]         at java.lang.reflect.Method.invoke(Method.java:498)
[ERROR]         at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:289)
[ERROR]         at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:229)
[ERROR]         at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:415)
[ERROR]         at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:356)

```

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -147,3 +147,4 @@ To accept, please:
 | Marcin Słowiak               |                                              | Sanok, Poland                           | 2019-09-23 |
 | David Barda                  |                                              | Tel-Aviv, Israel                        | 2019-10-14 |
 | Bishnu Gopal Patro           |                                              | Pune, India                             | 2019-10-19 |
+| Bartosz Czyzowski            |                                              | Jozefow, Poland                         | 2019-11-17 |

--- a/strongbox-web-core/src/test/java/org/carlspring/strongbox/forms/configuration/RepositoryFormTestIT.java
+++ b/strongbox-web-core/src/test/java/org/carlspring/strongbox/forms/configuration/RepositoryFormTestIT.java
@@ -371,8 +371,7 @@ public class RepositoryFormTestIT
         String message;
         try
         {
-            final String translatedMessage = new PlatformResourceBundleLocator(
-                    "org.hibernate.validator.ValidationMessages")
+            final String translatedMessage = new PlatformResourceBundleLocator("org.hibernate.validator.ValidationMessages")
                                                                  .getResourceBundle(Locale.getDefault())
                                                                  .getString("javax.validation.constraints.Pattern.message");
             message = translatedMessage.replace("\"{regexp}\"", expectedPattern);
@@ -381,6 +380,7 @@ public class RepositoryFormTestIT
         {
             message = "must match ";
         }
+        
         return message;
     }
 

--- a/strongbox-web-core/src/test/java/org/carlspring/strongbox/forms/configuration/RepositoryFormTestIT.java
+++ b/strongbox-web-core/src/test/java/org/carlspring/strongbox/forms/configuration/RepositoryFormTestIT.java
@@ -369,12 +369,15 @@ public class RepositoryFormTestIT
     private String getPatternLocalisedMessage(String expectedPattern)
     {
         String message;
-        try {
+        try
+        {
             final String translatedMessage = new PlatformResourceBundleLocator("org.hibernate.validator.ValidationMessages")
                     .getResourceBundle(Locale.getDefault())
                     .getString("javax.validation.constraints.Pattern.message");
             message = translatedMessage.replace("\"{regexp}\"", expectedPattern);
-        } catch (MissingResourceException e) {
+        }
+        catch (MissingResourceException e)
+        {
             message = "must match ";
         }
         return message;

--- a/strongbox-web-core/src/test/java/org/carlspring/strongbox/forms/configuration/RepositoryFormTestIT.java
+++ b/strongbox-web-core/src/test/java/org/carlspring/strongbox/forms/configuration/RepositoryFormTestIT.java
@@ -371,9 +371,10 @@ public class RepositoryFormTestIT
         String message;
         try
         {
-            final String translatedMessage = new PlatformResourceBundleLocator("org.hibernate.validator.ValidationMessages")
-                    .getResourceBundle(Locale.getDefault())
-                    .getString("javax.validation.constraints.Pattern.message");
+            final String translatedMessage = new PlatformResourceBundleLocator(
+                    "org.hibernate.validator.ValidationMessages")
+                                                                 .getResourceBundle(Locale.getDefault())
+                                                                 .getString("javax.validation.constraints.Pattern.message");
             message = translatedMessage.replace("\"{regexp}\"", expectedPattern);
         }
         catch (MissingResourceException e)

--- a/strongbox-web-core/src/test/java/org/carlspring/strongbox/forms/configuration/RepositoryFormTestIT.java
+++ b/strongbox-web-core/src/test/java/org/carlspring/strongbox/forms/configuration/RepositoryFormTestIT.java
@@ -12,10 +12,13 @@ import javax.inject.Inject;
 import javax.validation.ConstraintViolation;
 import javax.validation.Validator;
 import javax.validation.groups.Default;
+import java.util.Locale;
+import java.util.MissingResourceException;
 import java.util.Set;
 import java.util.stream.Stream;
 
 import org.apache.commons.lang3.StringUtils;
+import org.hibernate.validator.resourceloading.PlatformResourceBundleLocator;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.params.ParameterizedTest;
@@ -187,7 +190,7 @@ public class RepositoryFormTestIT
         repositoryForm.setHttpConnectionPool(HTTP_CONNECTION_POOL_VALID);
         repositoryForm.setRepositoryConfiguration(repositoryConfiguration);
 
-        validateAndAssert(repositoryForm, 1, "must match \"[a-zA-Z0-9\\-\\_\\.]+\"");
+        validateAndAssert(repositoryForm, 1, getPatternLocalisedMessage("\"[a-zA-Z0-9\\-\\_\\.]+\""));
     }
 
     @ParameterizedTest
@@ -362,4 +365,19 @@ public class RepositoryFormTestIT
 
         validateAndAssert(repositoryForm, 1, "A httpConnectionPool must be positive or zero.");
     }
+
+    private String getPatternLocalisedMessage(String expectedPattern)
+    {
+        String message;
+        try {
+            final String translatedMessage = new PlatformResourceBundleLocator("org.hibernate.validator.ValidationMessages")
+                    .getResourceBundle(Locale.getDefault())
+                    .getString("javax.validation.constraints.Pattern.message");
+            message = translatedMessage.replace("\"{regexp}\"", expectedPattern);
+        } catch (MissingResourceException e) {
+            message = "must match ";
+        }
+        return message;
+    }
+
 }

